pkg/chartverifier/checks/checks.go

```diff
@@ -274,6 +281,31 @@ func ImagesAreCertified(opts *CheckOptions) (Result, error) {
 	return r, nil
 }
 
+func NoMissingRequiredAnnotations(opts *CheckOptions) (Result, error) {
```

Comment:
I think the check name should be something positive like: "RequiredAnnotationsPresent"